simapp/simd/main.go

```diff
@@ -3,12 +3,18 @@ package main
 import (
 	"os"
 
+	"github.com/cosmos/cosmos-sdk/server"
 	"github.com/cosmos/cosmos-sdk/simapp/simd/cmd"
 )
 
 func main() {
 	rootCmd, _ := cmd.NewRootCmd()
 	if err := cmd.Execute(rootCmd); err != nil {
-		os.Exit(1)
+		switch e := err.(type) {
```

Comment:
Should we log an error as well?

Reply:
Exit code + logging information already provided by the underlying goroutines should do it all.

Reply:
Current error output when received `SIGTERM` is like this:
```
Error: 143
Usage:
  chain-maind start [flags]

Flags:
  ...
```